packages/frontend-v2/components/Error/ErrorBoundary.tsx

```diff
@@ -1,22 +1,38 @@
-// import { ErrorBoundary as ReactErrorBoundary } from "react-error-boundary";
-import React, { PropsWithChildren } from "react";
-// import { logger } from "../../utils";
-// import { ClientSideError } from "./ClientSideError";
+import React, { Component, ErrorInfo } from "react";
 
-// const clientSideErrorHandler = (
-//   error: Error,
-//   { componentStack }: { componentStack: string }
-// ) => {
-//   logger.error(error.message, componentStack);
-// };
+interface IProps {
+  fallback: React.FC;
+  children: React.ReactNode;
+}
 
-export const ErrorBoundary: React.FC<PropsWithChildren> = ({ children }) => {
-  return (
-    // <ReactErrorBoundary
-    //   FallbackComponent={ClientSideError}
-    //   onError={clientSideErrorHandler}
-    // >
-    <div>{children}</div>
-    // </ReactErrorBoundary>
-  );
-};
+interface IState {
+  hasError: boolean;
+}
+
+export class ErrorBoundary extends Component<IProps, IState> {
+  state = { hasError: false };
+
+  constructor(props: IProps) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error: Error) {
+    // Update state so the next render will show the fallback UI.
+    console.error(error);
```

Comment:
This is needed mainly for the signature of the error boundary I believe because the signature has this property but the parameter cant be passed if I don't use the console error.